math/binomialcoefficient_test.go

```diff
@@ -0,0 +1,39 @@
+// binomialcoefficient_test.go
+// description: Returns C(n, k)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see binomialcoefficient.go
+
+package math_test
+
+import (
+	"testing"
+
+	algmath "github.com/TheAlgorithms/Go/math"
```

Comment:
Why not just import it as is without `algmath` when there is no naming conflict now?